vmm/src/cpu.rs

```diff
@@ -1368,6 +1368,10 @@ impl Snapshottable for CpuManager {
             let vcpu = self
                 .create_vcpu(cpu_id.parse::<u8>().unwrap(), None, Some(*snapshot.clone()))
                 .map_err(|e| MigratableError::Restore(anyhow!("Could not create vCPU {:?}", e)))?;
+
+            // Adding vCPU to the CpuManager's vCPU list.
+            self.vcpus.push(vcpu.clone());
+
```

Comment:
Shouldn't this be part of create_vcpu instead?

Reply:
We can, but if we do so, we have to modify the "boot from scratch" codepath.